src/main/java/org/stellar/sdk/xdr/AccountMergeResult.java

```diff
@@ -58,4 +60,17 @@ public static AccountMergeResult decode(XdrDataInputStream stream) throws IOExce
   }
     return decodedAccountMergeResult;
   }
+  @Override
+  public int hashCode() {
+    return Objects.hash(this.code, this.sourceAccountBalance);
+  }
+  @Override
+  public boolean equals(Object object) {
+    if (object == null || !(object instanceof AccountMergeResult)) {
+      return false;
+    }
+
+    AccountMergeResult other = (AccountMergeResult) object;
+    return Objects.equals(this.sourceAccountBalance, other.sourceAccountBalance) && Objects.equals(this.code, other.code);
```

Comment:
It seems surprising that this generated code is coming up with a different order for the equals vs the arguments  in the hashCode. Other generated files I randomly sampled seem to have the same order.

Reply:
@ire-and-curses I pushed a commit to https://github.com/stellar/xdrgen/pull/46 which fixes this ordering issue and the `Objects.hash()` issue that @bartekn  found